src/Select/Select.component.tsx

```diff
@@ -190,14 +190,21 @@ const SDiv = styled.div<Props>`
       props.borderRadius || props.theme.common[props.selectSize!].borderRadius};
     border-color: ${(props: Props) =>
       props.borderColor || props.theme.select.borderColor || 'inherit'};
-	}
+  }
 `;
 
 const SSelect = styled(Select)`
+  font-family: ${(props: Props) => props.theme.typography.fontFamily};
   &:disabled {
     background: ${(props: Props) => props.theme.input.backgroundDisabled};
     cursor: not-allowed;
   }
+  react-select__value-container {
```

Comment:
should this be `.react-select__value-container`, with a dot at the beginning?

Reply:
good eye